test/e2e/chainsaw/operator/test-scenarios/stdio/test.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
```

Comment:
Why not do e2e tests like we do e2e tests for docker? That is, using golang's ginkgo